common/djangoapps/labster/masters.py

```diff
@@ -203,6 +204,8 @@ def get_problem_as_platform_xml(problem):
     }
 
     extra = {}
+    if problem.voice_id:
+        extra['VoiceId'] = str(problem.voice_id)
```

Comment:
Just to be consistent to the surrounding context. More over, the data is an attribute value and is not meant to be a translatable content.